miner/miner.go

```diff
@@ -83,8 +83,8 @@ type Config struct {
 
 // DefaultConfig contains default settings for miner.
 var DefaultConfig = Config{
-	GasCeil:  45_000_000,
-	GasPrice: big.NewInt(params.Wei),
+	GasCeil:  60_000_000,
+	GasPrice: big.NewInt(params.GWei / 1000),
```

Comment:
Our diff to the `GasPrice` is deliberate. I think it's because OP Stack miners no (1 wei) min gas price in general.

```suggestion
	GasCeil:  60_000_000,
	GasPrice: big.NewInt(params.Wei),
```

This setting ultimately translates into the default miner setting of the op-geth app, see https://github.com/ethereum-optimism/op-geth/blob/0b2dd7a070f63a0c3a0941bd2000b9bb1110c3a6/cmd/utils/flags.go?plain=1#L598-L603

Reply:
Ah I hit this in the monorepo PR and ended up changing it there. Will do it the other way round as you suggest.